assets/runtime/config/gitlabhq/gitlab.yml

```diff
@@ -478,6 +478,7 @@ production: &base
     # introduced in 9.0). Eventually Gitaly use will become mandatory and
     # this option will disappear.
     enabled: {{GITALY_ENABLED}}
+    token:
```

Comment:
This should have a placeholder as well `GITALY_TOKEN` . For the value it could be blank.